src/raven.js

```diff
@@ -1386,8 +1386,21 @@ Raven.prototype = {
     return frames;
   },
 
-  _normalizeFrame: function(frame) {
-    if (!frame.url) return;
+  _normalizeFrame: function(frame, stackInfoUrl) {
+    if (!frame.url) {
+      // Case when we don't have any information about the error
+      // E.g. throwing a string or raw object in Firefox
+      // Generating synthetic error doesn't add any value here
+      //
+      // We should probably somehow let user know that he should fix his code
```

Comment:
should say "they should fix their code"

Reply:
in_app and filename are the only changed keys here, right? would it be simpler to just make the `normalized` object assign `filename : frame.url || stackInfoUrl`, and rely on the existing logic for in_app? or does in_app actually return a false positive in this case?

Reply:
Correct. Just wanted to be a little more explicit, as we don't have function nor filename which are used in tests for in_app logic. I still left it on a separate line now though, as it's more readable with this long comment this way.